### DISCUSSION TITLE
Verdadeiros Fãs

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#303] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/verdadeiros-fas/

**Comment by one-eyed-willy** [comment]
![null](<https://i.postimg.cc/rp3n5VvS/giphy.webp>)

**Comment by brunoaugusto** [comment]
Esperando fanboyzada aparecer dando chilique sobre a frase do Saitama XD
